enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/messaging/ReconnectingChannel.java

```diff
@@ -95,7 +94,9 @@ else if ( fChannel != null && !fChannel.isDone() )
         {
             if ( !f.isSuccess() )
             {
-                f.channel().eventLoop().schedule( this::tryConnect, connectionBackoff.getMillis(), MILLISECONDS );
+                long millis = connectionBackoff.getMillis();
+                log.warn( "Failed to connect to: %s. Retrying in %d ms", destination.socketAddress(), millis );
```

Comment:
This will spam like crazy. Consider a CappedLogger with an interval of at least 10 seconds.